ros/src/twist_controller/cfg/DynReconf.cfg

```diff
@@ -8,8 +8,10 @@ PACKAGE = "twist_controller"
 
 gen = ParameterGenerator()
 
-gen.add("dyn_velo_proportional_control", double_t, 0, "Velocity proportional control constant", 0.1, 0, 2)
-gen.add("dyn_velo_integral_control", double_t, 0, "Velocity integral control constant", 0.000005, 0, 0.1)
-gen.add("dyn_velo_differential_control", double_t, 0, "Velocity integral control constant", 0.05, 0, 1)
+gen.add("dyn_velo_proportional_control", double_t, 0, "Velocity proportional control constant", 0.5, 0, 4)
+gen.add("dyn_accel_proportional_control", double_t, 0, "Acceleration proportional control constant", 1.0, 0, 4.0)
+gen.add("dyn_accel_integral_control", double_t, 0, "Acceleration integral control constant", 0, 0, 1)
```

Comment:
I found in practice (and had my findings [reinforced](https://www.controleng.com/single-article/fixing-pid-part-3/e115731163ee0db5abba1ade50cf4281.html)) that using non-zero integral constants caused throttle hunting when arriving just above the target velocity. As I noted in code with a TODO, I want to try to let the vehicle "coast" to a target velocity without braking in a future PR, but for now, I've got things working properly.